src/daemon/abrtd.c

```diff
@@ -435,7 +435,24 @@ static gboolean handle_signal_cb(GIOChannel *gio, GIOCondition condition, gpoint
 {
     uint8_t signo;
     gsize len = 0;
-    g_io_channel_read_chars(gio, (void*) &signo, 1, &len, NULL);
+    GError *error = NULL;
+    GIOStatus stat = g_io_channel_read_chars(gio, (void*) &signo, 1, &len, &error);
+    if (stat == G_IO_STATUS_ERROR)
+    {
+        /* An error occurred on the channel. Report it and quit. */
+        error_msg_and_die(_("Can't read from gio channel: '%s'"),
```

Comment:
Oh, right, we don’t want this to be translatable, since it’s not user-facing.

Reply:
Will fix.